certified-connectors/OptiAPI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,398 @@
+{
+  "basePath": "/api/v1/",
+  "consumes": ["application/json"],
+  "definitions": {},
+  "host": "optiapi.com",
+  "info": {
+    "contact": {
+      "email": "suppport@parsewire.com",
+      "name": "Oliver Busk Jensen"
+    },
+    "description": "The official api for OptiAPI, which connects countless of everyday actions with Power Automate.",
+    "license": {
+      "name": "MIT"
+    },
+    "title": "OptiAPI",
+    "version": "1.0.0"
+  },
```

Comment:
@oliverbj Could you please add back the x-ms-api-annotation to info section (like below)

```
  "info": {
    "contact": {
      "email": "suppport@parsewire.com",
      "name": "Oliver Busk Jensen"
    },
    "description": "The official api for OptiAPI, which connects countless of everyday actions with Power Automate.",
    "license": {
      "name": "MIT"
    },
    "title": "OptiAPI",
    "version": "1.0.0",
    "x-ms-api-annotation": {
      "status": "Preview"
    }
```